runtimes/polimec/src/lib.rs

```diff
@@ -211,7 +216,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("polimec-mainnet"),
 	impl_name: create_runtime_str!("polimec-mainnet"),
 	authoring_version: 1,
-	spec_version: 0_007_006,
+	spec_version: 0_008_000,
 	impl_version: 0,
 	apis: RUNTIME_API_VERSIONS,
 	transaction_version: 4,
```

Comment:
Last thing: we should also bump the transaction_version

![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/xPvTQnLOCB6vOZ5uHdYW/598ab060-9ebf-415d-be47-bcde6993d008.png)